Remove SORT_REGULAR flag from unique rule

dvdheiden opened the pull request:
Closes https://github.com/Respect/Validation/issues/1533

henriquemoody commented:
Thank you for your pull request, @dvdheiden!

A few changes before we merge your code. Could you add some tests to cover the new changes and write the reason for the change in the body of the commit message?

dvdheiden commented:
Hi @henriquemoody

I've added the testcase from the issue to the tests, see: https://github.com/Respect/Validation/pull/1534/files#diff-d86b663dae78d131902ff8c8552a2df8fdf484cb3f882aacd0616e3dbfb0493aR38 which covers this specific problem.

I've added the commit message body as you requested.

henriquemoody commented:
Thank you so much! However, it seems like you have 3 commits now instead of 1 with the description, I think you will need to rebase your branch

dvdheiden commented:
Done

henriquemoody commented:
I'm sorry for delaying this for so long, I have been quite busy.

I created another pull request because the tests on this one were failing. I had to make lots of changes to make everything work as expected.

henriquemoody commented:
Thank you for contributing! 🐼 

P.S: I'm sorry for overwriting your pull request, but I thought it would be best to see what I could make of it, and as it turned out, the problem was quite complicated.

dvdheiden commented:
No problem, especially because of your (understandable) explanation. Thank you for your work on this package!
